topbin/coqnative_bin.ml

```diff
@@ -322,4 +322,6 @@ let () =
     let senv = init_coq () in
     compile senv ~in_file
   with exn ->
-    Format.eprintf "Error: @[%a@]@\n%!" Pp.pp_with (CErrors.print exn)
+    Format.eprintf "Error: @[%a@]@\n%!" Pp.pp_with (CErrors.print exn);
+    let exit_code = if (CErrors.is_anomaly exn) then 129 else 1 in
```

Comment:
Has this been discussed somewhere? The convention on Unix is that exit code 128+N means that the signal N caused an abrupt termination of the process. So, using 129 might confuse shells, scripts, etc.

Reply:
This is the convention we already use for other coq binaries.

Reply:
This seems to date back from a while ago https://github.com/coq/coq/commit/78793236fa289d6e48cb10c9694ee493e0952c28